docs/integration_resources/software/nuke.md

```diff
@@ -14,59 +14,107 @@ nshift.show()
+|:-------------|:---------------|:-------------------------------------------------|:-------------------------------------------------------------|:----------------------------------------------------------------------------------------------------|
+| *Bool*       | -              | Knob                                             | Boolean_Knob                                                 |                                                                                                     |
+| *Color*      | -              | Knob                                             | AColor_Knob                                                  |                                                                                                     |
+| *Dict*       | -              | Knob                                             | EvalString_Knob                                              | Require custom implementation                                                                       |
+| *Dir*        | -              | Knob                                             | File_Knob                                                    |                                                                                                     |
+| *Enumerator* | -              | Knob                                             | Enumeration_Knob                                             |                                                                                                     |
+| *FileIn*     | -              | Knob                                             | File_Knob                                                    |                                                                                                     |
+| *FileOut*    | -              | Knob                                             | File_Knob                                                    |                                                                                                     |
+| *Float*      | -              | Knob                                             | Double_Knob                                                  |                                                                                                     |
+| *Instance*   | image*         | Input/Output<br/>Knob<br/>Knob                   | - <br/>Channel_Knob <br/>File_Knob                           | Provides the input image like a single image. (Current Frame)                                       |
+| *Instance*   | images*        | Output<br/>Knob<br/>Knob                         | - <br/>File_Knob                                             | Provides the input image like a single image. (Current Frame)                                       |

```

Comment:
Why images is only output??

Let's open a conversation on this durnig a meeting.

Reply:
After discussing this is set as output only because the "images" are lists of images which are not time correlated and for this reason we do not intend the user to use them as inputs.

This type of plug is useful for ease of displaying multiple examples produced by a workflow.